script/Ciel/Day 9/09_01_CIEL09_2B.txt

```diff
@@ -1153,7 +1153,7 @@ Their abilities have always been a part of nature――no, it's more that they a
 -- Page 224, Offset 22705.
 -- <真祖|かれら>は世界そのものから力を引き上げる。<真祖|かれら>を倒すということは世界を倒すほどの概念武装が必要になります。
 True Ancestors derive their power from the planet itself.
- Therefore, the only Conceptual Weapons capable of taking them down would be those capable of taking down the entire world.
+ Therefore, the only Conceptual Weapons capable of taking them down would be those capable of taking down the entire planet.
```

Comment:
I'd prefer to have them both be world.

Reply:
I feel like a lot of the lore uses planet, does it not make sense to be consistent about that?